### PR TITLE
Add justification for classifier based Python interpreter version check

### PR DESCRIPTION
**Pull request by fridex**
Related: https://github.com/thoth-station/thoth-application/issues/1850

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: _j/trove_py.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: No Python specific trove classifier matching Python version used
+---
+
+Package maintainer did not state Python version used as a compatible version.
```
Comment (KPostOffice) [inline _j/trove_py.md:5]:
```suggestion
Python package version used is not listed in the list of compatible versions by the package maintainer.
```

**Review by KPostOffice — COMMENTED** [review]
One comment

File: _j/trove_py.md (hunk 1 above)
Comment (fridex) [reply _j/trove_py.md]: Thanks, applied 👍🏻

**Pull request merged**